yt_dlp/extractor/cda.py

```diff
@@ -351,3 +351,63 @@ def extract_format(page, version):
             extract_format(webpage, resolution)
 
         return merge_dicts(info_dict, info)
+
+
+class CDAFolderIE(InfoExtractor):
+    _MAX_PAGE_NUMBER = 200
+    _VALID_URL = r'https?://(?:www\.)?cda\.pl/[0-9a-zA-Z_]+/folder/(?P<id>[0-9a-z]+)(/vfilm)?'
+    _TESTS = [
+        {
+            'url': 'https://www.cda.pl/domino264/folder/31188385',
+            'info_dict': {
+                'id': '31188385',
+                'title': 'SERIA DRUGA',
+            },
+            'playlist_count': 13,
```

Comment:
Do we expect new videos to be added here? If yes use `playlist_mincount`

Reply:
I was looking for old and not updated playlists to be sure this number won't change